src/macrofier/engine.mjs

```diff
@@ -175,12 +175,43 @@ const getMethods = compose(
   getPath(['methods'])
 )
 
-const getSchemas = compose(
+const getComponentSchemas = compose(
   map(Object.entries), // Maybe Array<Array<key, value>>
   chain(safe(isObject)), // Maybe Object
   getPath(['components', 'schemas']) // Maybe any
 )
 
+const getSchemas = (schema) => {
```

Comment:
Instead of doing this for all the languages, can we do this based on a config option? namely anonymousSchemas which when set to true for a language, processes these method parameter types separately? We should leave the getSchemas as it was if the config is not set.

Reply:
added config option to choose anonymous schema based on language